roles/download/defaults/main.yml

```diff
@@ -830,6 +830,8 @@ etcd_image_repo: "{{ quay_image_repo }}/coreos/etcd"
 etcd_image_tag: "{{ etcd_version }}{%- if image_arch != 'amd64' -%}-{{ image_arch }}{%- endif -%}"
 flannel_image_repo: "{{ quay_image_repo }}/coreos/flannel"
 flannel_image_tag: "{{ flannel_version }}-{{ image_arch }}"
+flannel_init_image_repo: "rancher/mirrored-flannelcni-flannel-cni-plugin"
```

Comment:
The offline deployment feature cannot work if container images contain static repo names like `rancher/` because the repo name parts(`{{ quay_image_repo }}` for example) are replaced with local repos for offline deployment.
Related to @cristicalin's comment, can we use a common repo for that instead of the static `rancher/` ?

Reply:
Ok, I've replaced it to docker.io/flannelcni/flannel-cni-plugin:v1.0.0

Reply:
I don't quite understand the comment in https://github.com/flannel-io/flannel/blob/349feae40ae4c8a57c5abbaf9c4fcfe43cadc264/Documentation/kube-flannel.yml#L169-L170, what docker hub limitations affect `ppc64le`? I'm asking this since we now support ppc64le arch and we should be aware (and maybe document) the limitation.

Reply:
Now Kubespray is using flannel v1.0.0 and the flannel-cni-plugin image on the dockerhub supports
```
linux/amd64
linux/arm/v6
linux/arm64/v8
linux/ppc64le
linux/s390x
```
as https://hub.docker.com/r/flannelcni/flannel-cni-plugin/tags
It is better to investigate more to know the reason of the comment.

Reply:
I think I see the reason of the above comment.
https://github.com/flannel-io/flannel/issues/1482#issuecomment-944356060 shows
- The dockerhub flannel-cni-plugin is the official image
- The rancher flannel-cni-plugin is for avoiding a rate limit issue due to the dockerhub policy
- `image: flannelcni/flannel-cni-plugin:v1.0.1 for ppc64le (dockerhub limitations may apply)` means the rancher flannel-cni-plugin doesn't support ppc64le and we need to enable the dockerhub one instead for ppc64le if we are using.

To support ppc64le also, it is necessary to use the dockerhub image.
If using the dockerhub image, a rate limit issue could happen as the above comment.
However Kubespray already is using a lot of dockerhub images, I don't feel that can be a matter for us.